images/sqlpad/tests/03-helm-test.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+cd "${SCRIPT_DIR}/../"
+
+TMPDIR="$(mktemp -d)"
+
+git clone https://github.com/sqlpad/sqlpad.git "${TMPDIR}/sqlpad"
+
+# Using local patch till postgresql 13.2.24 gets fixed in upstream charts https://github.com/sqlpad/sqlpad/blob/f42ce1a84e1327e69178f38e7b3a2462dcaeeb7c/sqlpad-charts/requirements.yaml#L3
+# A PR is also up for that https://github.com/sqlpad/sqlpad/pull/1212
+cp patches/helm.patch "${TMPDIR}/sqlpad"
```

Comment:
nit (non-blocking, can be addressed in the future): this could be:
```bash
cp ${SCRIPT_DIR}/patches/helm.patch ...
```
so that it works even when the script is invoked from a different directory.